experiments/long_runs/snowy_land.jl

```diff
@@ -49,6 +49,9 @@ import NCDatasets
 
 using Poppler_jll: pdfunite
 
+using CSV, DataFrames, HTTP, Flux, StatsBase, cuDNN, BSON
```

Comment:
I would suggest reverting this file back (git checkout origin/main experiments/long_runs/snowy_land.jl), because for us to merge this, we need to check if the long runs are stable for multiple years with this addition.